models/Group.model.js

```diff
@@ -0,0 +1,33 @@
+const { Schema, model } = require("mongoose");
+
+const groupSchema = new Schema(
+  {
+    groupName: {
+      type: String,
+      require: true,
+      unique: true
+    },
+    description: {
+      type: String,
+    },
```

Comment:
tipp: you can write it in one line: 

```suggestion
description: String
```